{"name": "登录到首页", "status": "passed", "steps": [{"name": "1、点击X，关闭设置密码弹窗", "status": "passed", "start": 1656647035975, "stop": 1656647038029}, {"name": "2、在首页点击测评量表", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"app\"]/div/div[2]/div/div/div/div[1]/div/div[2]/p[1]\"}\n  (Session info: chrome=103.0.5060.66)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x009BD953+2414931]\n\tOrdinal0 [0x0094F5E1+1963489]\n\tOrdinal0 [0x0083C6B8+837304]\n\tOrdinal0 [0x00869500+1021184]\n\tOrdinal0 [0x0086979B+1021851]\n\tOrdinal0 [0x00896502+1205506]\n\tOrdinal0 [0x008844E4+1131748]\n\tOrdinal0 [0x00894812+1198098]\n\tOrdinal0 [0x008842B6+1131190]\n\tOrdinal0 [0x0085E860+976992]\n\tOrdinal0 [0x0085F756+980822]\n\tGetHandleVerifier [0x00C2CC62+2510274]\n\tGetHandleVerifier [0x00C1F760+2455744]\n\tGetHandleVerifier [0x00A4EABA+551962]\n\tGetHandleVerifier [0x00A4D916+547446]\n\tOrdinal0 [0x00955F3B+1990459]\n\tOrdinal0 [0x0095A898+2009240]\n\tOrdinal0 [0x0095A985+2009477]\n\tOrdinal0 [0x00963AD1+2046673]\n\tBaseThreadInitThunk [0x7620FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77BB7A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77BB7A4E+238]\n\n", "trace": "  File \"D:\\work\\pythonProject(1)(1)\\pythonProject\\case\\6noble.py\", line 49, in test_02\n    report_text=wk.get_text(REPORT_TEXT)\n  File \"D:\\work\\pythonProject(1)(1)\\pythonProject\\KeyWords\\keyword_web.py\", line 68, in get_text\n    return self.locator(\"xpath\", path).text\n  File \"D:\\work\\pythonProject(1)(1)\\pythonProject\\KeyWords\\keyword_web.py\", line 44, in locator\n    return self.driver.find_element(name, value)\n  File \"C:\\Users\\jay52\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 1251, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Users\\jay52\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 430, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\jay52\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1656647038029, "stop": 1656647049119}], "attachments": [{"name": "log", "source": "d2a6354f-b4c8-4070-af71-aa6a6cf8fad9-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "88c9a76b-7cf9-402b-815b-a834411e54d3-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a32bd3ce-dc10-48d3-b9fc-ccedae902c4c-attachment.txt", "type": "text/plain"}], "start": 1656647035960, "stop": 1656647049135, "uuid": "5197fba1-89db-4d93-bb53-0dab40c8fa12", "historyId": "2998ce8627b489f8392e858ad587f5cb", "testCaseId": "cde3bcf5d1d4de99cd06364941980dd4", "fullName": "6noble#test_02", "labels": [{"name": "suite", "value": "6noble"}, {"name": "host", "value": "DESKTOP-FA1JTDA"}, {"name": "thread", "value": "22004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "6noble"}]}